stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/logic/ChannelLogic.kt

```diff
@@ -97,6 +100,11 @@ internal class ChannelLogic(
             }
     }
 
+    override suspend fun onChannelMarkReadPrecondition(channelType: String, channelId: String): Result<Unit> =
```

Comment:
Missing docs

Reply:
It's an implementation. This function is documented in the interface so omitting here.